frontend/projects/upgrade/src/app/features/dashboard/feature-flags/modals/upsert-feature-flag-modal/upsert-feature-flag-modal.component.ts

```diff
@@ -234,10 +247,13 @@ export class UpsertFeatureFlagModalComponent {
     this.featureFlagsService.addFeatureFlag(flagRequest);
   }
 
-  createEditRequest(
-    { name, key, description, appContext, tags }: FeatureFlagFormData,
-    { id, status, filterMode }: FeatureFlag
-  ): void {
+  createEditRequest({ name, key, description, appContext, tags }: FeatureFlagFormData, sourceFlag: FeatureFlag): void {
+    const { id, status, filterMode } = sourceFlag;
```

Comment:
Is this change needed when you are already disabling the form?

Reply:
disabled fields aren't available with `form.value` so I did it this way.